src/core/thread.d

```diff
@@ -5676,6 +5676,9 @@ private
     }
 }
 
+version (Windows)
+package(core) bool _thread_dll_process_detaching;
```

Comment:
Why snake case instead of camel case and why the leading underscore?

Reply:
I had this variable in core.sys.windows.dll first which uses this coding style. Now changed it to `thread_DLLProcessDetaching` following the style of `thread_attachThis`.